.vscode/cspell.json

```diff
@@ -127,6 +115,145 @@
         "policheck",
         "gdnbaselines"
       ]
+    },
+    {
+      "filename": "sdk/videoanalyzer/video-analyzer-edge/review/**/*.md",
+      "words": [
+        "abgr",
+        "Abgr",
+        "argb",
+        "Argb",
+        "bgra",
+        "Bgra",
+        "Grpc",
+        "onvif",
+        "Onvif"
+      ]
+    },
+    {
+      "filename": "sdk/storage/storage-blob/review/**/*.md",
+      "words": [
+        "RAGRS",
+        "Uncommited"
+      ]
+    },
+    {
+      "filename": "sdk/search/search-documents/review/**/*.md",
+      "words": [
+        "Adls",
+        "adlsgen",
+        "bangla",
+        "Bokmaal",
+        "Createor",
+        "Decompounder",
+        "haase",
+        "koelner",
+        "kstem",
+        "kstem",
+        "lovins",
+        "nysiis",
+        "odatatype",
+        "Phonetik",
+        "Piqd",
+        "reranker",
+        "Rslp",
+        "sorani",
+        "Sorani"
+      ]
+    },
+    {
+      "filename": "sdk/monitor/monitor-query/review/**/*.md",
```

Comment:
Do we want to ignore this word in all files that could end up under `/monitory-query/review/**/*.md` or just ignore this word in a single file?

Reply:
@danieljurek I read your comment about scoping actual spelling errors to the actual files they're in, which will prevent repeating the error somewhere else in the same directory (In this case, under `/monitory-query/review/`). 

However, my understanding is that there's only 1 .md file inside all `<package-name>/review/` directories in the repo. If you believe that they should be scoped to a single file regardless, let me know and I can make the necessary changes 👍🏼

Reply:
I am aware of only one exception: https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/monitor/monitor-opentelemetry-exporter/review

I'm not sure how these exception happened... If this will not happen in your folder then the `**/*.md` is fine.

Reply:
@ramya-rao-a true but making the change will not make a difference in the behavior with the current structure.

What I'm going to do is to scope the 6 actual spell errors to only the files they're in, so if in the future we will have more `.md` files or even subfolders under `review/` these errors won't be allowed.

Reply:
I am absolutely ok with scoping the errors to the files they are in